template/module/report/name.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
The QWeb report (prints) templates are now in `views`, the `report folder being reserved to BI/analysis reports: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#directories